### PR TITLE
Adla

### PR DESCRIPTION
**Pull request by AdlaMostafa**

### CODE REVIEW

File: src/components/pages/HomePage/Components/atoms/FormSelect/FormSelect.jsx
Comment (se-bayan-elian) [inline src/components/pages/HomePage/Components/atoms/FormSelect/FormSelect.jsx:1]: the form select is already defined in atoms by me , if u use this for home page just , modify the name of Component , to don't refer it as global

File: src/components/pages/HomePage/Components/atoms/FormTextarea/FormTextarea.jsx
Comment (se-bayan-elian) [inline src/components/pages/HomePage/Components/atoms/FormTextarea/FormTextarea.jsx:1]: unusefull , what the needs to make this component u can directly uses the text area

File: src/components/pages/HomePage/Components/molecules/ProductCategoryComponent/style.js
Comment (se-bayan-elian) [inline src/components/pages/HomePage/Components/molecules/ProductCategoryComponent/style.js:1]: the style for Typographys not needed ,just use the component

File: src/components/pages/HomePage/Components/molecules/ProductCategoryComponent/ProductCategoryComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import Typography from '../../../../../atoms/Typography'
+import Image from 'next/image'
+import { StyledProductCategoryComponent } from './style'
+import { StyledP } from '../../../../../atoms/Typography/style'
+const ProductCategoryComponent = ({title,price,src}) => {
+  return (
+    <StyledProductCategoryComponent>
+        <div>
+           <div className='title'><h5>{title}</h5></div>
```
Comment (se-bayan-elian) [inline src/components/pages/HomePage/Components/molecules/ProductCategoryComponent/ProductCategoryComponent.jsx:10]: <Typography as="p" variant="default-muted" ></...>
not exact variant ,,, 
// variant for p 
    // default => 16 px 
    // default-muted => 16px color:muted
    // small => 14 px 
    // small-muted => 14px color:muted
    // micro => 12 px 
    // micro-muted => 12px color:muted

File: src/components/pages/HomePage/Sections/DealsAndOffersSection/style.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    margin-bottom:-5px;
+    font-family: 'Inter', sans-serif;
+  }
+}
+img{
+  width:150px;
+  height:130px;
+}
+p{
+  font-family: 'Inter', sans-serif;
```
Comment (se-bayan-elian) [inline src/components/pages/HomePage/Sections/DealsAndOffersSection/style.js:42]: u don't need to add font-family in any component else , all of them are common "Inter"

File: src/components/pages/HomePage/Sections/ProductsByCategorySection1/ProductsByCategorySection1.jsx
Comment (se-bayan-elian) [inline src/components/pages/HomePage/Sections/ProductsByCategorySection1/ProductsByCategorySection1.jsx:1]: u don't need to add section 1 and to u can resolve it by using map on mock data like this : 
const data = {
section1 :  data fot it ... ,
section2 : data fot it 
}
and render them in page directly

File: src/components/pages/HomePage/Components/organism/ProductsByCategorySection/index.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import ProductCategoryComponent from '../../../Components/molecules/ProductCategoryComponent/ProductCategoryComponent';
+import Typography from '../../../../../atoms/Typography';
+
+const ProductsByCategorySection = ({ productData = [], background,doubleItemContent}) => {
+  return (
+    <GridParent>
+      <DoubleSizeItem style={{backgroundImage:`url(${background})`}}>
+        <div className='DoubleItemContent'>
+         <h2 className='doubleItem-title'>{doubleItemContent?.title}</h2>
+        <button>{doubleItemContent?.buttonText}</button></div>
```
Comment (mo7mad4) [inline src/components/pages/HomePage/Components/organism/ProductsByCategorySection/index.jsx:12]: Hello Adla , 
I hope this message finds you well,
this line button is pure but we make Button component you can use it here
this code is very good in line 
```js
        <Button variant='secondary' >{doubleItemContent?.buttonText}</Button></div>
```
and also you can import Buttom from Component folder

File: src/components/pages/HomePage/index.jsx [hunk 4]
```diff
@@ -1,8 +1,28 @@
 import React from 'react'
```
Comment (mo7mad4) [inline src/components/pages/HomePage/index.jsx:1]: In this line make ```"use client";```
coz , we make styled component

File: src/components/pages/HomePage/Components/atoms/FormSelect/FormSelect.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
```
Comment (mo7mad4) [inline src/components/pages/HomePage/Components/atoms/FormSelect/FormSelect.jsx:1]: In this line 1 make ```"use client";```
coz , we make styled component

**Pull request merged**